package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@salesforce/salesforce-sdk",
+  "name": "sf-fx-sdk-nodejs",
   "version": "2.0.0",
   "description": "Salesforce SDK for Node.js",
-  "main": "src/index.js",
+  "main": "src/index.ts",
```

Comment:
What is the reason for changing `main`? This seemed correct before, since `main` should refer to the built assets (ie: output of the TypeScript compiler, since one would publish as JS to NPMJS).

Reply:
youre right, the output of the tsc compiler actually is in ./dist so I changed `main` and `types` to point there.